swagger-ga4gh-client/pom.xml

```diff
@@ -0,0 +1,243 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>io.swagger</groupId>
+  <artifactId>swagger-ga4gh-client</artifactId>
+  <packaging>jar</packaging>
+  <name>swagger-ga4gh-client</name>
+  <version>1.0.0</version>
+
+  <parent>
+    <version>2.0.0-alpha.16-SNAPSHOT</version>
+    <groupId>io.consonance</groupId>
+    <artifactId>consonance</artifactId>
+    <relativePath>../pom.xml</relativePath>
+  </parent>
+
+  <url>https://github.com/swagger-api/swagger-codegen</url>
+
+  <description>Swagger Java</description>
+  <scm>
+    <connection>scm:git:git@github.com:swagger-api/swagger-codegen.git</connection>
```

Comment:
probably should strip these paths since they're not applicable.

Reading further, I think much of this content is generated, but shouldn't be kept since it will make its way to your released artifacts